backend/models/projects/projects-model.test.js

```diff
@@ -1,8 +1,24 @@
 // const mongoose = require("mongoose")
-// const projectsModel = require("./projects-model")
+const ProjectsModel = require("./projects-model")
 const db = require("../db-test-setup")
 
 // Define test instances here
+const testProject1 = {
+    title: "Project 1",
+    status: "In Progress",
+    description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, " +
+        "sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. " +
+        "Ut enim ad minim veniam, quis nostrud exercitation ullamco " +
+        "laboris nisi ut aliquip ex ea commodo consequat. ",
+    start_date: new Date("12-24-2022"),
+    end_date: new Date("12-24-2023"),
+    discord_url: "https://discord.gg/12345",
+    github_url: "github.com/12345",
```

Comment:
Make sure add a valid url
```suggestion
    github_url: "https://github.com/12345",
```